<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>77</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addProjFile">
      <property name="text">
       <string>添加文件</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delProjFile">
      <property name="text">
       <string>删除文件</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+D</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="saveConfigAs">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>301</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveConfig">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>301</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>301</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveResult">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>301</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存解析结果</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
